### PR TITLE
Add included services

### PR DESCRIPTION
**Pull request by dati91**

### CODE REVIEW

File: components/script/dom/bluetoothremotegattservice.rs [hunk 1]
```diff
@@ -155,4 +156,59 @@ impl BluetoothRemoteGATTServiceMethods for BluetoothRemoteGATTService {
...
+        let services_vec = receiver.recv().unwrap();
+        match services_vec {
+            Ok(service_vec) => {
+                Ok(service_vec.into_iter()
+                              .map(|service| BluetoothRemoteGATTService::new(self.global().r(),
+                                                                             &self.device.get(),
+                                                                             DOMString::from(service.uuid),
+                                                                             service.is_primary,
+                                                                             service.instance_id))
+                              .collect())
```
Comment (mmatyas) [inline components/script/dom/bluetoothremotegattservice.rs:207]: szegény kis collect itt árválkodik egymaga :)

**Comment by mmatyas** [comment]
r+

File: components/net/bluetooth_thread.rs [hunk 2]
```diff
@@ -478,6 +485,66 @@ impl BluetoothManager {
...
+                                  });
+            }
+        }
+        if let Some(uuid) = uuid {
+            services_vec.retain(|ref s| s.uuid == uuid);
+        }
+        if services_vec.is_empty() {
+            return drop(sender.send(Err(String::from(INCLUDED_SERVICE_ERROR))));
+        }
+
```
Comment (mmatyas) [inline components/net/bluetooth_thread.rs:544]: no need for this empty line

Comment (dati91) [inline components/net/bluetooth_thread.rs:544]: I just followed the other functions' style

**Comment by zakorgy** [comment]
r+

File: tests/html/bluetooth_included_service_info.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            var device = window.navigator.bluetooth.requestDevice(options);
+
+            log('Connecting to GATTserver on device...');
+            var server = device.gatt.connect();
+
+            log('Getting Primary Service...');
+            var primaryService = server.getPrimaryService(filterService);
+
+            log('Primary Service found on device: ' + primaryService.device.name);
+
```
Comment (mmatyas) [inline tests/html/bluetooth_included_service_info.html]: remove this empty line

**Pull request merged**